pkg/command/command.go

```diff
@@ -19,13 +19,13 @@ type Command struct {
 	env    []string
 }
 
-// Dir sets the working directory for the execution
-func (c *Command) Dir(d string) {
+// SetDir sets the working directory for the execution
```

Comment:
This change is not related to the scope of the pull request which is "Move mock capabilities ..."